.github/workflows/release.yml

```diff
@@ -13,37 +13,55 @@ env:
   HUSKY: 0
 
 jobs:
-  release:
-    concurrency: ${{ github.workflow }}-${{ github.ref }}
+  get-branch-name:
+    runs-on: ubuntu-latest
+    outputs:
+      current_branch: ${{ steps.branch-name.outputs.current_branch }}
+    steps:
+      - name: Get branch name
+        id: branch-name
+        uses: tj-actions/branch-names@v7.0.7
+  check-dialpad-member:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [ 20 ]
-
     steps:
-      - uses: actions/checkout@v3
-      - uses: pnpm/action-setup@v2
+      # Will prevent the rest of the steps from running on fail
+      - name: Check if user is a dialpad member
+        uses: octokit/request-action@v2.x
         with:
-          version: 8
+          route: GET /orgs/dialpad/members/${{ github.actor }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.DIALTONE_CI_TOKEN }}
```

Comment:
Don't think we actually need this anymore right? These should be limited to only run by those with admin / maintainer permissions

Reply:
Really, we're we set that? I don't remember setting that up.

Reply:
Hmm I think it's maybe just running the local release command that doesn't let you do it if you're not admin. Anyways don't worry about it for now I will look into it some more.